api/urls.py

```diff
@@ -61,6 +61,12 @@
     DashboardPersonalInfoViewSet,
     basename="personal_info",
 )
+dashboard_router.register(
+    r"languages_and_levels", LanguageAndLevelViewSet, basename="languages_and_levels"
+)
+dashboard_router.register(
+    r"day_and_time_slots", DayAndTimeSlotViewSet, basename="day_and_time_slots"
+)
```

Comment:
Probably wee need to create a separate view set , e.g: `DashboardDayAndTimeSlotViewSet`, just to be consistent

Reply:
hmm, not sure where the difference would actually be, the dashboard needs the exact same format of data and the non-dahsboard views are already read-only which is also what we want

Reply:
Currently there is not difference so it is just to follow the same design approach as we did everywhere. But it might in a different PR of course

Reply:
Well @dont-text-me since you created this PR to be consistent with the logic of separation of bot-oriented URLs from dashboard-oriented ones, then to me it would be only logical to follow this approach in the viewsets as well. Otherwise it will be a sort of half-PR :)

Reply:
fair enough 😄